rulesets/standard.json

```diff
@@ -60,7 +60,7 @@
     "//planned-disbursement/provider-org | //planned-disbursement/receiver-org | //transaction/provider-org | //transaction/receiver-org | //recipient-org-budget/recipient-org": {
         "regex_matches": {
             "cases": [
-                { "regex": "^(?!\s*$).+",
+                { "regex": "^\s*\S.*",
```

Comment:
Linked to the code comment in https://github.com/IATI/IATI-Rulesets/pull/57/files it appears that this is not a valid JSON string:

```
>>> json.loads('{ "regex": "^\s*\S.*"}')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Invalid \escape: line 1 column 13 (char 13)
```

Reply:
It appears the fix to this issue is to only reference a subset of regex syntax described here: https://spacetelescope.github.io/understanding-json-schema/reference/regular_expressions.html#regular-expressions

Reply:
Looks like you can just escape the backslash with another backslash. There are other examples of this in the same file e.g. here:
https://github.com/IATI/IATI-Rulesets/blob/c33d11885e1084bb99d897c9a5dc3bfcea91c0fb/rulesets/standard.json#L19

Reply:
The syntax highlighting on github suggests it works.

Bad:
```json
{
  "bad": "\O\h\ \d\e\a\r"
}
```

Fine:
```json
{
  "fine": "\\O\\K\\A\\Y"
}
```